kitsune/users/jinja2/users/edit_profile.html

```diff
@@ -51,28 +53,9 @@ <h1 class="sumo-page-heading">{{ title }}</h1>
       <div>
         <section class="avatar-group">
           <figure class="avatar-group--figure">
-            <img class="avatar" src="{{ profile_avatar(profile.user) }}" alt="">
-            {% if not profile.is_fxa_migrated %}
-              <a class="change-avatar text-body-sm display-block" href="https://gravatar.com/emails" target="_blank">
-                {{ pgettext('avatar', 'Choose Photo') }}
-                <span class="tip">
-                  {% trans %}
-                  <p>
-                    SUMO is now using Gravatars as avatars. You must change
-                    your Gravatar on gravatar.com.
-                  </p>
-                  <p>
-                    It can take a few minutes for the avatar change to take
-                    effect. You may need to wait a few minutes and reload
-                    the page to see your new Gravatar.
-                  </p>
-                  {% endtrans %}
-                </span>
-              </a>
-              {% if profile.avatar %}
-                <a class="remove-avatar" href="{{ url('users.delete_avatar') }}">{{ _('Delete') }}</a>
-              {% endif %}
-            {% endif %}
+            <label>Avatar
+              <img class="avatar" src="{{ profile_avatar(profile.user) }}" alt="">
```

Comment:
IMO we need a bit of spacing between the label and img, looks a bit cramped currently.